tests/unit/htdocs/BannersSecurityTest.php

```
@@ -0,0 +1,219 @@
+
+    #[Test]
+    public function sessionStoresClientIdNotPassword(): void
+    {
+        // Simulate what the login handler now does
+        $_SESSION = [];
+        $clientId = 42;
+        $_SESSION['banner_client_id'] = (int) $clientId;
+
+        $this->assertSame(42, $_SESSION['banner_client_id']);
+        $this->assertArrayNotHasKey('banner_pass', $_SESSION);
+    }
```

PHPMD (2.15.0)

[error] sessionStoresClientIdNotPassword accesses the super-global variable $_SESSION. (undefined)

(Superglobals)